libs/wingsdk/.projenrc.ts

```diff
@@ -29,9 +29,7 @@ const EX_DOCS_PREFIX = "../../docs/docs/04-standard-library/02-ex/";
 
 // defines the list of dependencies required for each compilation target that is not built into the
 // compiler (like Terraform targets).
-const TARGET_DEPS = {
-  awscdk: ["aws-cdk-lib@^2.64.0"],
-};
+const TARGET_DEPS: { [key: string]: string[] } = {};
```

Comment:
I debated on removing this side car loading entirely, but I figured that if its needed in the future its would not be totally clear how to do it. So I opted to leave it in as empty. 

If someone objects and wants it deleted, you wont get any push back from me